tests/specs/opcodes/create-summary-spec.k

```diff
@@ -0,0 +1,175 @@
+requires "verification.k"
+
+/*
+
+`CREATE VALUE MEMSTART MEMWIDTH` will return (push into the stack) `#newAddr(ACCT_ID, NONCE)`, updating the network state as follows:
+
+```
+<network>
+  <activeAccounts> SetItem(ACCT_ID) ACCTS:Set (.Set => SetItem(#newAddr(ACCT_ID, NONCE))) </activeAccounts>
+  <accounts>
+    <account>
+      <acctID> ACCT_ID </acctID>
+      <balance> BAL => BAL -Int VALUE </balance>
+      <nonce> NONCE => NONCE +Int 1 </nonce>
+      ...
+    </account>
+  (
+    .Bag
+  =>
+    <account>
+      <acctID> #newAddr(ACCT_ID, NONCE) </acctID>
+      <balance> VALUE </balance>
+      <nonce> 1 </nonce>
+      <code> #generatedCode </code>
+      ...
+    </account>
+  )
+    ...
+  </accounts>
+</network>
```

Comment:
Remove this duplicate, since it could become out of sync with the proof itself. Leave the comment for explanation.